components/MashCards/ScreeningCard.tsx

```diff
@@ -10,7 +10,19 @@ interface Props {
 const ScreeningCard = ({ mashReferral }: Props): React.ReactElement => {
   return (
     <>
-      <li className={s.row}>
+      {' '}
```

Comment:
No ' ' for spacing please.

Also think it is tidier to use && instead of a ternary when false just renders to an empty string.